hudi-common/src/main/java/org/apache/hudi/common/config/DFSPropertiesConfiguration.java

```diff
@@ -113,15 +112,14 @@ public static TypedProperties loadGlobalProps() {
       }
     }
     // Try loading the external config file from local file system
+    try {
+      conf.addPropsFromFile(DEFAULT_PATH);
+    } catch (Exception e) {
+      LOG.warn("Cannot load default config file: " + DEFAULT_PATH, e);
+    }
```

Comment:
I think this makes sense so the default configs are always loaded and user can provide a config file to override ones of interest, instead of putting all default configs in the user-provided config file.

Reply:
We'll need to update the docs to reflect this change.

Reply:
+1. The change looks fine to me, but it calls for a doc update as it changes the expected behavior: the `DEFAULT_CONF` would always be applied unless overriden by some user conf

Reply:
Created HUDI-8223 for docs update.

Reply:
Docs is also updated in `ConfigGroups` class.